src/main/resources/com/sonyericsson/jenkins/plugins/bfa/model/FailureCauseMatrixBuildAction/summary.groovy

```diff
@@ -33,31 +33,38 @@ def l = namespace(lib.LayoutTagLib)
 
 index = 1
 
-tr {
-    td {
-        img(width: "48", height: "48", src: my.getImageUrl(), style: "margin-right:1em;")
-    }
-    td(style: "vertical-align: middle;") {
-        h2(_("Identified problems"))
-    }
-}
```

Comment:
From what I can see the default text will never be displayed?

Reply:
It will when the conditional below is true (line 37); the block reappears on line 41 indented in, the unified diff does not make it clear, try to give a look with side-by-side view

Reply:
Ah yes, see my comment in `FailureCauseBuildAction/summary.groovy` :)